eval.py

```diff
@@ -189,7 +193,7 @@ def compute(gold_dataset, guess_dataset, ks=None, inv_propensity_scores_dict=Non
     for guess, gold in zip(guess_dataset, gold_dataset):
         try:
             assert (
-                str(gold["uid"]).strip() == str(guess["id"]).strip()
+                str(gold["id"]).strip() == str(guess["id"]).strip()
```

Comment:
will this break the compatibility with other datasets used in GROOV?

Reply:
Good point! I will add support for both identifiers :-).